website/screens/theme-generator/themes/opinionated-theme.json

```diff
@@ -64,6 +64,9 @@
   "link": {
     "baseColor": "#5f249f"
   },
+  "navTabs": {
+    "baseColor": "#5f249f"
+  },
```

Comment:
I would remove this until the opinionated theme is done.

Reply:
removed!